### PR TITLE
store: log store for report snapshot.

### PR DESCRIPTION
**Pull request by siddontang**
@ngaut @BusyJay @disksing

### CODE REVIEW

**Comment by BusyJay** [comment]
LGTM

File: src/raftstore/store/store.rs [hunk 1]
```diff
@@ -842,8 +842,10 @@ impl<T: Transport, C: PdClient> Store<T, C> {
 
     fn on_report_snapshot(&mut self, region_id: u64, to_peer_id: u64, status: SnapshotStatus) {
         if let Some(mut peer) = self.region_peers.get_mut(&region_id) {
-            info!("report to snapshot {} for {} {:?}",
-                  to_peer_id,
+            // The peer must be in peer_cache.
+            let to_peer = self.peer_cache.rl().get(&to_peer_id).cloned().unwrap();
+            info!("report to snapshot {:?} for {} {:?}",
```
Comment (ngaut) [inline src/raftstore/store/store.rs]: s/to snapshot/snapshot status/

File: src/raftstore/store/store.rs [hunk 2]
```diff
@@ -842,8 +842,10 @@ impl<T: Transport, C: PdClient> Store<T, C> {
 
     fn on_report_snapshot(&mut self, region_id: u64, to_peer_id: u64, status: SnapshotStatus) {
         if let Some(mut peer) = self.region_peers.get_mut(&region_id) {
-            info!("report to snapshot {} for {} {:?}",
-                  to_peer_id,
+            // The peer must be in peer_cache.
```
Comment (ngaut) [inline src/raftstore/store/store.rs]: s/be/exists/

**Comment by ngaut** [comment]
LGTM

**Pull request merged**